src/vulkan-renderer/application.cpp

```diff
@@ -194,23 +196,34 @@ void Application::load_shaders() {
 void Application::load_octree_geometry() {
     spdlog::debug("Creating octree geometry.");
 
-    std::shared_ptr<world::Cube> cube =
-        std::make_shared<world::Cube>(world::Cube::Type::OCTANT, 2.0f, glm::vec3{0, -1, -1});
-
-    cube->childs()[3]->set_type(world::Cube::Type::EMPTY);
-    cube->childs()[5]->set_type(world::Cube::Type::EMPTY);
-    cube->childs()[6]->set_type(world::Cube::Type::EMPTY);
-    cube->childs()[7]->set_type(world::Cube::Type::EMPTY);
-
-    for (const auto &polygons : cube->polygons(true)) {
-        for (const auto &triangle : *polygons) {
-            for (const auto &vertex : triangle) {
-                glm::vec3 color = {
-                    static_cast<float>(rand()) / static_cast<float>(RAND_MAX),
-                    static_cast<float>(rand()) / static_cast<float>(RAND_MAX),
-                    static_cast<float>(rand()) / static_cast<float>(RAND_MAX),
-                };
-                m_octree_vertices.emplace_back(vertex, color);
+    // Let's just create 2 worlds for testing now.
```

Comment:
3 worlds?

Reply:
That's wrong: we create 3. It is already fixed :)
We can create as many as we like of course, but comments should match code.